Mobile/iOS/Templates/Release_3.5.0/648-txt_documents.md

```diff
@@ -0,0 +1,31 @@
+###  Txt documents 
+
+#### Pr: https://github.com/owncloud/ios/pull/648 
+
+Devices:
+
+Server:
+
+---
+
+ 
+| TestID | Test Case | Steps | Expected Result | Result | Related Comment |
+| :----: | :-------- | :---- | :-------------- | :----: | :------ |
+| **View** |||||| 
+| 1 | New file option - Portrait  | 1. Tap on (...) with the device in Portrait | The option "New txt file" is shown |  |  |
+| 2 | New file option - Landscape  | 1. Tap on (...) with the device in Landscape | The option "New txt file" is shown |  |  |
+| 3 | New file Editor - Portrait  | 1. In portrait, tap on (...) and select "New txt file"<br>2. Save a file  | File can be created, written and saved correctly |  |  |
+| 4 | New file Editor - Landscape  | 1. In landscape, tap on (...) and select "New txt file"<br>2. Save a file  | File can be created, written and saved correctly |  |  |
+| **File** |||||| 
+| 5 | Empty file name | 1. Tap on (...) and select "New txt file"<br> 2. Try to save the file with a empty name | The file can not be saved |  |  |
+| 6 | Special Characters  | 1. Tap on (...) and select "New txt file"<br> 2. Try to save the file whose name contains special characters | The file is saved correctly |  |  |
+| 7 | Long file name  | 1. Tap on (...) and select "New txt file"<br> 2. Try to save the file whose name is too long (+50characters) | The file is saved correctly |  |  |
+| 8 | Duplicated file name (same folder) | 1. Tap on (...) and select "New txt file"<br> 2. Try to save the file with a existent file name in the current folder| A conflict is detected and the file can not be saved |  |  |
+| 9 | Duplicated file name (different folder) | 1. Tap on (...) and select "New txt file"<br> 2. Try to save the file with a existent file name in the a different folder| The file is correctly saved |  |  |
```

Comment:
It's my fault... bad explanation. The aim of test case #9 is saving a txt file with the same name as other one existing in a different folder to check there is no conflict although the target folder are not the same.

Fixing... 👍